modules/System Admin/import_run.php

```diff
@@ -97,7 +97,7 @@
         $availableModes = array();
         $modes = $importType->getDetail('modes');
         if (!empty($modes['update']) && !empty($modes['insert'])) {
-            $availableModes['sync'] = __('Update').' & '.__('Insert');
+            $availableModes['sync'] = __('Update & Insert');
```

Comment:
This possibly creates a new string for translation, but definitely more elegant!

Reply:
Hi Ross! This string already existed! :clap: